src/gui/update.rs

```diff
@@ -1876,13 +1876,22 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
                 state.accounts = accounts;
 
                 // If we have an account already selected, use that as the picklist selection
+                // or if there is only a single account to choose, use that
                 // and trigger a parse for this without user interaction
-                if let Some(account) = ajour.config.weak_auras_account.get(&flavor) {
+                let account_from_config = ajour.config.weak_auras_account.get(&flavor).cloned();
+                let get_single_account = || {
+                    if state.accounts.len() == 1 {
+                        Some(state.accounts[0].clone())
```

Comment:
Maybe a little prettier to use `.first()` (https://doc.rust-lang.org/std/vec/struct.Vec.html#method.first)?

Reply:
`.first()` here is a bit awkward since it returns an Option.

It would be convenient if there existed an 'exactly one' function that returned the first element if it was the only element.

Writing this made me realise that maybe this is a job for pattern matching

Reply:
Yeah, thats true. Otherwise I am fine with using this for now.